package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
```

Comment:
I thought `clean` should get you back to a pristine working tree?

I can re-add it for you, but you end up with this:

```
$ npm run clean

> emoji-regex@6.5.1 clean /Users/tim/Codez/emoji-regex
> rm -rf -- index.js text.js es2015

$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	deleted:    es2015/index.js
	deleted:    es2015/text.js
	deleted:    index.js
	deleted:    text.js

no changes added to commit (use "git add" and/or "git commit -a")
```

That's different to what a normal `npm run clean` or `make clean` would be expected to do (actually delete source files)?

Your call!